wiki/osu!wiki_contribution_guide/en.md

```diff
@@ -1,936 +1,80 @@
-<!-- Internal -->
-[ASG]: /wiki/Article_Style_Guide "Article Style Guide"
-[GitHub osu-wiki]: https://github.com/ppy/osu-wiki "osu-wiki in GitHub"
-[GH Issue]: https://github.com/ppy/osu-wiki/issues "osu-wiki Issues page"
+osu!wiki Contribution Guide
+========================
 
-<!-- External -->
-[osu!dev Discord]: https://discord.gg/ppy "osu!dev in Discord"
-[MD]: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet "adam-p's cheatsheet for Markdown"
-[ISO 639-1]: https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes "List of ISO 639-1 codes in Wikipedia"
-[E325 error]: https://stackoverflow.com/questions/14513349/e325-attention-found-a-swap-file-by-the-name-git-commit-editmsg-swp "E325 question in Stack Overflow"
-[Matt Liu image source]: http://www.slideshare.net/MatthewKLiu/git-branch-management "Git branch management by Matt Liu"
-[Git set default editor]: https://swcarpentry.github.io/git-novice/02-setup/ "swcarpentry on setting up the default editor"
-[Githowto resolve conflicts]: https://githowto.com/resolving_conflicts "Githowto's 30. Resolving Conflicts"
-[git mergetool help by karenyyng]: https://gist.github.com/karenyyng/f19ff75c60f18b4b8149 "karenyyng's git mergetool guide"
-[Ruslan Osipov's vimdiff tutorial]: http://www.rosipov.com/blog/use-vimdiff-as-git-mergetool/ "Ruslan Osipov's vimdiff tutorial"
-[GH Blog GPG]: https://github.com/blog/2144-gpg-signature-verification "GitHub blog on GPG signature verification"
-[GPG guide by Duncan]: http://mrexodia.cf/guide/2016/05/30/Github-Gpg "mrexodia blog post on GitHub GPG by Duncan"
+This quick-start guide is intended for newcomers that have never worked with GitHub and the wiki contribution workflow before. If you already have experience using GitHub, feel free to skip this guide and follow the common feature-branch workflow for content updates in this repository.
```

Comment:
not sure about `have experience using GitHub`
I would change that `using` into `with` or `in`; probably wait for Eph